frontend/lendme/lib/screens/other/add_item.dart

```diff
@@ -1,26 +1,206 @@
+import 'dart:io';
+import 'package:firebase_storage/firebase_storage.dart';
 import 'package:flutter/material.dart';
+import 'package:cloud_firestore/cloud_firestore.dart';
+import 'package:firebase_auth/firebase_auth.dart';
+import 'package:image_picker/image_picker.dart';
+import 'package:permission_handler/permission_handler.dart';
 
-class AddItem extends StatelessWidget {
+class AddItem extends StatefulWidget {
   const AddItem({Key? key}) : super(key: key);
 
+  @override
+  _AddItemState createState() => _AddItemState();
+}
+
+class _AddItemState extends State<AddItem> {
+  final FirebaseAuth firebaseAuth = FirebaseAuth.instance;
+  CollectionReference items = FirebaseFirestore.instance.collection('items');
+  String description = '';
+  String itemName = '';
+  var imageUrl = null;
+  var localImage;
+  bool isLoading = false;
+
   @override
   Widget build(BuildContext context) {
+    Future getImage() async {
+      final _storage = FirebaseStorage.instance;
+      File _image;
+      final _picker = ImagePicker();
```

Comment:
Leading underscore says that variable is private, however these variables are local, so being private is even not applicable for them. Remove underscores or make them class fields if you want.